teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -100,14 +117,157 @@ requirements, and provenance attestations can be made to meet software supply ch
+
+Resources start in the Community Catalog, then they may get promoted to the Official Catalog upon meeting
+the requirements if the Tekton Catalog Maintainers approve. If a resource is promoted to the Official
+Catalog, it should be [deprecated][deprecation] in the Community Catalog.
+
+#### Tekton Community Catalog
+
+To maintain the low barrier to making contributions, users can easily submit resources to the Community Catalog.
+
+The current Tekton Catalog at `tektoncd/catalog` will be reused as the Community Catalog.
+
+##### Requirements
+
+1. The resource must be structured per the contract in [TEP-0003: Tekton Catalog Organization][tep-0003-org].
+1. The resource must define an OWNER file in `/{resource-type}/{resource-name}/OWNERS`, that specifies at least one 
+  maintainer for that specific resource.
+
+#### Tekton Official Catalog
+
+To provide a corpus of high quality resources that users can rely on, we propose creating an Official Catalog 
+with high maintenance, testing and security standards. 
+
+The Official Catalog will be in a new repository named `tektoncd/catalog-official`.
+
+##### Requirements
+
+1. The resource must be structured per the contract in [TEP-0003: Tekton Catalog Organization][tep-0003-org].
+1. The resource must not define an OWNER file, given that it is owned and maintained by Tekton Catalog Maintainers who 
+  are defined at the root of the repository. The ownership responsibilities are described [below](#responsibilities).
+1. The resource must have automated testing using Tekton and, preferably, used in dogfooding. If there are any failures, 
+  they must be resolved as soon as possible; the SLO is one week. The automated testing is discussed further [below](#automated-testing-and-dogfooding-1).
+1. The resource must be scanned for common vulnerabilities and exposures. If any issues are discovered, they must be 
+  patched or disclosed as soon as possible; the SLO is one week. The scanning for CVEs is discussed further [below](#image-scanning-for-common-vulnerabilities-and-exposures-cves-1).
+1. The resource must support verification as proposed in [TEP-0091: Verified Remote Resources][tep-0091]. 
+  For now, it must be published to a public OCI registry as a [Tekton Bundle][bundle] and signed by Tekton. 
+  In the future, it must be updated to support accepted proposal of TEP-0091 depending on the direction of 
+  [TEP-0060: Remote Resource Resolution][tep-0060]. The verification is discussed further [below](#verified-remote-resources-1).
+1. The resource must be well documented with all configuration options described and a working example provided.
+1. The resource must follow and demonstrate best practices, such as the [`Task` authoring recommendations][task-authoring-recommendations].
+
+#### Responsibilities
+
+There are two types of ownership: resource ownership and catalog ownership.
+
+The responsibilities of owning a resource include:
```

Comment:
Do we need to specify "routine maintenance" like maintaining up-to-date versions of dependencies?

Reply:
agreed, added it to the forth item:

> 4. Updating the resources to be compatible with new versions of Tekton, and other dependencies.